client/client_bean.go

```diff
@@ -22,7 +22,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-//go:generate mockgen -copyright_file ../LICENSE -package $GOPACKAGE -source $GOFILE -destination client_bean_mock.go
+//go:generate go run ../cmd/tools/mockgen/main.go -copyright_file ../LICENSE -package $GOPACKAGE -source $GOFILE -destination client_bean_mock.go
```

Comment:
You can leave out the `main.go` if there's just a `main()` func in a `main` package
```suggestion
//go:generate go run ../cmd/tools/mockgen -copyright_file ../LICENSE -package $GOPACKAGE -source $GOFILE -destination client_bean_mock.go
```